docs/source/caching/cache-key-resolution.mdx

```diff
@@ -8,181 +8,66 @@ The normalized cache computes a **cache key** for each object that is stored in
 
 > To learn more, read about how the cache [normalizes objects by cache key](./introduction#normalizing-objects-by-cache-key).
 
-## `CacheKeyInfo`
+For most uses cases, you can configure cache keys declaratively using the methods described here. For advanced use cases not supported by declarative cache keys, you can configure cache keys [programmatically](./programmatic-cache-keys).
```

Comment:
Might just be me but clicking "programmatic cache keys" in the preview makes a 404 (link is https://www.apollographql.com/docs/deploy-preview/1f96ea7a044024b699863083/ios/caching/programmatic-cache-keys)